src/Filters/response.jl

```diff
@@ -58,6 +60,24 @@ function phasez(filter::FilterCoefficients, w = range(0, stop=π, length=250))
 end
 
 
+"""
+    grpdelay(fliter, w = range(0, stop=π, length=250))
+
+Group delay of a digital 'filter' at normalized frequency
+or frequencies 'w' in radians/sample.
+"""
+function grpdelay(filter::FilterCoefficients, w = range(0, stop=π, length=250))
+    filter = convert(PolynomialRatio, filter)
+    b, a = coefb(filter), filter.a.coeffs # reversed a
+    c = conv(b, conj(a))
+    cr = range(0, stop=length(c)-1) .* c
+    ejw = exp.(-im .* w)
+    num = Polynomial(cr).(ejw)
+    den = Polynomial(c).(ejw)
+    gd = real(num ./ den) .- length(a) .+ 1
```

Comment:
```suggestion
    return real.(num ./ den) .- (length(a) - 1)
```
The `return` for stylistic reasons, the `.` to avoid unnecessary temporaries by allowing broadcast fusion, and the `- 1` to do the subtraction only once instead of once per element. We could avoid further allocations by substituting `num` and `den` with their definitions, but that might be at the cost of readability, not sure if worth it.